lib/libutee/tee_api_objects.c

```diff
@@ -451,7 +451,7 @@ TEE_Result __GP11_TEE_PopulateTransientObject(TEE_ObjectHandle object,
 }
 
 void TEE_InitRefAttribute(TEE_Attribute *attr, uint32_t attributeID,
-			  const void *buffer, uint32_t length)
+			  const void *buffer, size_t length)
```

Comment:
My first approach was to avoid wrappers for these simple cases, however, I discovered that it didn't work if the TA uses function pointers for certain things. So instead of trying to figure out which functions matter and which don't, I'm trying to do all of them as well as possible.

Another advantage of this approach is that it doesn't clutter the tee_internal_api.h file too much.